src/ArduinoIoTCloudTCP.cpp

```diff
@@ -426,9 +425,7 @@ ArduinoIoTCloudTCP::State ArduinoIoTCloudTCP::handle_SendDeviceProperties()
     return State::Disconnect;
   }
 
-#if OTA_ENABLED
```

Comment:
Will this be a problem for devices which do not support OTA, i.e. MKR 1000, MKR GSM 1400, ... ?

Reply:
I think that also this line is outdated and not part of the final changes, but this made me doublecheck the code because after your question i was a bit unsure about it, so thanks.

In the final changes there are two versions of the `sendDevicePropertiesToCloud()` the first one sends all the device properties and it is availabe for all boards, the second one accept a string as input to send one property at time ad is used and availabe only for boards supporting OTA.